packages/eslint-config-nimble-testing/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@nimblehq/eslint-config-nimble-testing",
+  "version": "2.4.0",
+  "description": "ESLint testing plugin extension configuration developed and maintained by Nimble",
+  "keywords": [
+    "eslint",
+    "eslint-plugin",
+    "eslint-testing",
+    "eslint-jest",
+    "eslint-cypress",
+    "eslintplugin"
+  ],
+  "author": "Nimble",
+  "main": "lib/index.js",
+  "dependencies": {
+    "@nimblehq/eslint-config-nimble-core": "^2.4.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1"
+  },
+  "peerDependencies": {
+    "eslint": "^8.12.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-jest": "^27.2.1"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
+  },
+  "license": "MIT",
+  "directories": {
+    "lib": "lib"
+  },
+  "engines": {
+    "node": "^14.18.2 || ^16 || ^17"
```

Comment:
Should we rebase #46 on top of this PR, so that we can ensure that the version 18 will not be forgotten in this file (and maybe other dependencies 🤔 )

Reply:
Rebased and updated in [1af4364](https://github.com/nimblehq/eslint-config-nimble/pull/49/commits/1af43648757c88b92f31a824419b206ce635d8fa)